frontend/app/components/CardBack.tsx

```diff
@@ -36,48 +33,130 @@ export const CardBack = ({
         {
           width: CARD_WIDTH,
           height: CARD_HEIGHT,
-          backgroundColor: '#151312',
+          backgroundColor: '#1c1a2d',
         },
         backAnimatedStyle,
       ]}
     >
-      <Text style={styles.content}>{item.content}</Text>
+      <ScrollView
+        contentContainerStyle={styles.scrollContent}
+        showsVerticalScrollIndicator={false}
+        keyboardShouldPersistTaps="handled"
+      >
+        <Text style={styles.title}>{item.content}</Text>
+
+        <Animated.View style={styles.divider} />
+        <Section title="Meaning" content={item.meaning} />
+        <Section title="Explanation" content={item.explanation} />
+        <Section title="Examples" content={item.examples} />
```

Comment:
Section is a too general name for this component. Let's do one for each: `Meaning`, `Explanation`, `Examples`